include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -847,12 +847,17 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
         }
     }
 
-    auto needToCompMipsForThisImg = [](const asset::ICPUImage* img) -> bool {
-        if (img->getRegions().size() == 0u)
+    auto needToCompMipsForThisImg = [](const asset::ICPUImage* img) -> bool
+    {
+        if (img->getRegions().empty())
             return false;
         auto format = img->getCreationParameters().format;
         if (asset::isIntegerFormat(format) || asset::isBlockCompressionFormat(format))
             return false;
+        // its enough to define a single mipmap region above the base level to prevent automatic computation
+        for (auto& region : img->getRegions())
+        if (region.imageSubresource.mipLevel)
+            return false;
         return true;
     };
```

Comment:
just give this a look @Erfan-Ahmadi

Reply:
yea it makes sense, althought I would use  `> 0u` instead of treating it like a bool :D